lib/src/css_mirroring/transformer.dart

```diff
@@ -73,12 +75,20 @@ class BidiCssTransformer extends AggregateTransformer
 
     if (mapAsset != null) transform.consumePrimary(mapAsset.id);
 
-    var bidiCss = await bidirectionalizeCss(await cssAsset.readAsString(),
-        _flipCss, _settings.originalCssDirection.value);
-    if (_settings.verbose.value) {
-      transform.logger.info('Bidirectionalized css:\n$bidiCss');
+    if (_settings.flippingScheme.value == FlippingScheme.flip) {
```

Comment:
I'd use a switch, and throw in the default case (to make sure we don't collide cases if someone adds an enum value)